hash/msxr_flop.xml

```diff
@@ -239,11 +376,85 @@ Known undumped:
 		</part>
 	</software>
 
-	<software name="munote">
-		<description>μ.Note (Jpn)</description>
+	<software name="destiny2" supported="no">
+		<description>innocent wish ~destiny2~ (Japan)</description>
+		<year>1995</year>
+		<publisher>Sign House</publisher>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="innocent wish - destiny2 - sign house.dsk" size="737280" crc="ee349375" sha1="4abff41ef488885802c2d0056bbdc4a9d7d3e1c7"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="hoippuru" supported="no">
+		<description>Mahou no Kuni no Hoippuru (Japan)</description>
+		<year>1996</year>
+		<publisher>Pastel Hope</publisher>
+		<info name="alt_title" value="魔法の国のほいっぷる"/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="mahouno kunino hoippuru - pastel hope.dsk" size="737280" crc="e584c2f4" sha1="168c8ac3f2bf1cb327114b9a66bca99eed55f610"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="mechbrn" supported="no">
+		<description>Mechanical Brain (Japan)</description>
+		<year>1996</year>
+		<publisher>Studio Sequence</publisher>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="mechanical brain - studio sequence.dsk" size="737280" crc="61050e08" sha1="4f69e1ccb9e4622c0f0870c10fda4df00e01e97f"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="mejuusa" supported="no">
+		<description>Mejuu Isago - Medusa (Japan)</description>
+		<year>1994</year>
+		<publisher>Blue Eyes</publisher>
+		<info name="alt_title" value="眼獣・沙 ~メジューサ~"/>
+		<part name="flop1" interface="floppy_3_5">
```

Comment:
I’d like to see box art for this one to see if there’s an official Latin script title.  The pun in the title doesn’t translate well.

“眼獣” (Mejuu) is a compound of 眼 – eye/vision — and 獣 – beast/monster, as in “kaiju”.  “メジューサ” (Mejuusa) is a transliteration of Medusa (remember Japanese has no “du” sound as such, it’s pronounced as “zu”, so they’ve used “ju” instead, and lengthened it because it’s the stressed syllable).

The pun is that 眼獣 and the first three on of メジューサ are the same – “mejuu”.

Reply:
This was distributed through Brother's Takeru service so there is no box art. There is also no image of the receipt from the Takeru machines so we cannot see what title they used there.

I don't know if it helps, but this is what is shown during startup of the software:
![mejuusa](https://github.com/mamedev/mame/assets/9140441/82452a03-f980-4953-aa10-310e04e4e00d)

Reply:
Yeah, that’s just the Japanese title – it shows that in the heading throughout gameplay: http://rygames.sblo.jp/article/186065045.html